### ISSUE TITLE
Python version issue: import error

### ISSUE THREAD

**Issue by ilsa16** [LE2P/PyBsrnQC#3]
The latest python version compatible with the package (python 3.5.6) has other issues of itself. I try to import pybsrnqc but I get the following error:
Traceback (most recent call last):
  File "bsrn_qc.py", line 8, in <module>
    import pybsrnqc.config
  File "/home/ilsa16/anaconda3/envs/bsrn/lib/python3.5/site-packages/pybsrnqc/config.py", line 1, in <module>
    import importlib.resources
ImportError: No module named 'importlib.resources'

I installed a compatible pip version and installed the pybsrnqc. 
'importlib.resources' is compatible only for 3.7 or later.

**Comment by bilbaoba** [comment]
I suppose importlib is not installed on your python. 
Could you try to execute :

```sh
pip install importlib 
```
It seems it miss importlib in install_requires in the `confy.py` file of project.

**Comment by bilbaoba** [comment]
@ilsa16 did my proposal fix your issue ?

**Comment by ilsa16** [comment]
no, importlib is available for python >= 3.7

**Comment by bilbaoba** [comment]
Ok I see. I read about python 3.5 was launch in September 13th, 2015.  

I suggest you to fork the project and modify it to allow your old version of python working with pybsrnqc. 

The only part of code using imporlib is : 

```python
import importlib.resources
import json

# Load default conf from package
with importlib.resources.path("pybsrnqc", "qcrad_conf.json") as data_path:
    with open(data_path, 'r') as f:
        loaded_json = json.load(f)
```

It's just a way to import configurations.

I will mention on the package it work only with **Python >= 3.7**.
